### DISCUSSION TITLE
Can I fine tune a model with different setting from original?

### DISCUSSION THREAD

**Discussion by turbosonics** [MDIL-SNU/SevenNet#80] [Q&A]
I tried fine tune a model based on MP pretrained model, but it seems that I need to exactly follow some hyperparameters. Can I change while varying some hyperparameters? 

For example, I hope to increase cutoff to 6.0 and change other 'model' parameters such as radial basis while fine tune a model based on MP pretrained model, but that fine-tune training job crashes. 

Is this possible to change those numbers while fine tune?

**Comment by YutackPark** [comment]
As you already know SevenNet does not support such options and they are not planned. The reason is that, if we allow such modifications, most of those changes will eventually raise an error, as the number of parameters of the modified model is not the same as a pre-trained model. The cutoff you mentioned (or radial basis configured with not trainable) is likely the only exception. Still, it is expected to give a very large error at the beginning of fine-tuning.

If you want to experiment with it, remove the below line from the SevenNet code and give it a shot. You also need to rebuild your dataset if it is already preprocessed to `.sevenn_data`.
https://github.com/MDIL-SNU/SevenNet/blob/9c59ecebbae3e804bba1d19210e8af9499783e32/sevenn/scripts/processing_continue.py#L19
